package/src/renderer/Host.ts

```diff
@@ -16,14 +17,18 @@ export abstract class Node<P = unknown> {
   memoizable = false;
   memoized = false;
   parent?: Node;
+  depMgr: DependencyManager;
 
-  constructor(props: AnimatedProps<P>) {
+  constructor(depMgr: DependencyManager, props: AnimatedProps<P>) {
     this._props = props;
+    this.depMgr = depMgr;
+    this.depMgr.addValues(props);
   }
 
   abstract draw(ctx: DrawingContext): void | DeclarationResult;
 
   set props(props: AnimatedProps<P>) {
+    this.depMgr.addValues(props);
```

Comment:
Wondering if we need to unsubscribe first before subscribing? If we get property updates?